ui/src/app/shared/components/dropdown.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { DataLoader, Select, SelectOption } from 'argo-ui';
+
+export const Dropdown = (props: {load: Promise<(string | SelectOption)[]>; onChange: (value: string) => void}) => {
```

Comment:
Have about calling this `DataLoaderDropdown`?

Reply:
How about the filename?

Reply:
`data-loader-dropdown.tsx`

Reply:
updated to `DataLoaderDropdown`